code/headers/st7735.hpp

```diff
@@ -47,8 +48,8 @@ namespace r2d2::display {
         constexpr static uint8_t _GMCTRN1 = 0xE1;
 
         // since it uses a generic driver the display has a offset
-        constexpr static uint8_t x_offset = 26;
-        constexpr static uint8_t y_offset = 1;
+        constexpr static uint8_t x_offset = 0; //26
```

Comment:
Maybe a comment why these comments are here?

Reply:
Sure, I will add comments because the previous display module hardcoded offsets when in fact that differs per screen. The offset issue will not be resolved in this PR, as that is not part of the scope.

Reply:
Fixed in 5f9d398